config/locales/en.models.yml

```diff
@@ -274,6 +274,7 @@ en:
         is_active: "Is Active?"
       product:
         can_be_used_by?: Approved?
+        cross_facility_ordering_available: "Available for Cross-Facility Ordering?"
```

Comment:
```suggestion
        cross_facility_ordering_available: "Available for Cross-!Facility! Ordering?"
```

Reply:
If we need to call this "Cross-Resource Ordering" for Dartmouth, this is how we would do it